maquetado/scss/details.scss

```diff
@@ -1,48 +1,60 @@
-.book-details {
-  .book-item {
-    display: block;
-    font-size: $font-normal;
-    padding: 12px;
-  }
-  
-  .book-item-span {
-    color: $grey;
-    font-size: $font-normal;
-  }
-}
-
-.book-title {
-  padding: 24px 5px 10px;
-
-  .title-genre {
-    color: $grey;
-    font-size: $font-large;
-    font-weight: bold;
-  }
-}
-
-.title {
-  color: $black;
-  font-size: $font-extra-large;
-  font-weight: bold;
-  margin-right: 10px;
-}
-
 .book-card {
+  box-shadow: 5px 5px 10px $grey;
   display: flex;
   flex-flow: row;
+  font-family: 'Roboto', sans-serif;
   height: 100%;
   justify-content: flex-start;
   margin: 5px;
   max-height: 416px;
   max-width: 900px;
-  width:100%;
+  width: 100%;
 
   .book-image {
     height: 100%;
-    max-height: 368px;
-    max-width: 261px;
+    max-height: 416px;
+    max-width: 309px;
     padding: 24px;
     width: 100%;
   }
+
+  .book {
+    flex-grow: 1;
+    max-height: 368px;
+    max-width: 588px;
+    padding: 24px 24px 0px 24px;
+
+    .book-details {
+      .book-item {
+        display: block;
+        font-size: $font-normal;
+        padding: 36px 0 0 0 ;
```

Comment:
Change to padding-left: 36px

Reply:
But if you want to make a distance between the children and the parent, so add padding to book-details.

Reply:
This is not the case, that padding is to separate each paragraph (A great advice anyway). However, change the padding to padding-top.